src/components/FeaturedProjects.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ProjectCard from './ProjectCard';
+
+
+const FeaturedProjects = () => {
+    return (
+        <div className="m-8 md:w-6/12">
+            <h1 className="text-2xl md:text-left font-semibold mb-4">Featured Projects</h1>
+            <ProjectCard />
```

Comment:
make use of loops to create project cards. you can accept `projects` array as props to this component and use it to  create Project Card by iterating through it. for now can pass dummy data in array as props.